App/Redux/RootReducer.ts

```diff
@@ -354,14 +355,37 @@ const migrations: MigrationManifest = {
     const state = persistedState as any
     const { cameraRoll, group, uploadingImages, photos, ...rest } = state
     return rest
+  },
+  22: persistedState => {
```

Comment:
Unfortunately I haven't had a chance to test this since I factory reset my emulator while I was testing out the onboarding code. Do you remember what error message you were seeing?

Reply:
Nevermind, found the issue and fixed it. Let me know if you still encounter issues migrating.